web/cobrands/sass/_layout.scss

```diff
@@ -892,9 +892,6 @@ textarea.form-error {
 #front-recently h2 {
    margin-top:0;
 }
-#front-howto lite-youtube ~ h2 {
```

Comment:
Sorry if my previous comment wasn't clear - when I said "we don't want to remove this base style because eg Oxfordshire are also using it." I meant we don't want to be removing anything from this file.